src/app/(cord)/layout.tsx

```diff
@@ -19,21 +19,53 @@ export async function getData() {
     0,
     USERS.findIndex((user) => user.user_id === userIdCookie),
   );
-  const user = USERS[userIndex];
 
+  createAndPopulateGroup();
+  const user = USERS[userIndex];
   const clientAuthToken = getClientAuthToken(CORD_APP_ID, CORD_SECRET, {
     ...user,
-    // By supplying the `organization_details` object, just like the user,
-    // Cord will create the organization on-the-fly.
-    organization_details: {
-      name: user.organization_id,
-    },
   });
   return {
     clientAuthToken,
   };
 }
 
+/**
+ * Creates a group and adds all users to it.
+ *
+ * In a real app, you would do this only once.
+ **/
+async function createAndPopulateGroup() {
+  const { CORD_SECRET, CORD_APP_ID } = process.env;
+  if (!CORD_SECRET || !CORD_APP_ID) {
+    console.error(
+      "Missing CORD_SECRET or CORD_ORD_ID env variable. Get it on console.cord.com and add it to .env",
```

Comment:
```suggestion
      "Missing CORD_SECRET or CORD_APP_ID env variable. Get it on console.cord.com and add it to .env",
```
(this was a pre-existing issue I think)